hr_employee_phone_pin/models/hr_employee_public.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2024 Binhex (<https://binhex.cloud>)
+# Copyright 2024 Binhex Ariel Barreiros
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import api, fields, models
+
+
+class EmployeePublic(models.Model):
+    _inherit = "hr.employee.public"
+
+    mobile_phone_pin = fields.Char(
+        related="employee_id.mobile_phone_pin", readonly=True
+    )
+    mobile_phone_puk = fields.Char(
+        related="employee_id.mobile_phone_puk", readonly=True
+    )
+
+    is_current_user = fields.Boolean(compute="_compute_is_current_user")
+
+    @api.depends("user_id")
+    def _compute_is_current_user(self):
+        for rec in self:
+            rec.is_current_user = rec.user_id == self.env.user
```

Comment:
Is it necessary for these fields to be in the employee's public profile?

Reply:
I could remove the related property from pin and puk, but they are related to the ones on hr.employee anyway 😅. Also tried to remove them just to be sure, and it fails to install the app if they are not on the public model.
On the other hand is_current_user  is required to be on the view since pin and puk are only visible when is_current_user is true. This logic only happens on the public form